src/lib/Form/Factory/FormFactory.php

```diff
@@ -1327,4 +1333,49 @@ public function trashContainerLocation(
 
         return $this->formFactory->createNamed($name, LocationTrashContainerType::class, $data);
     }
+
+    /**
+     * @param \EzSystems\EzPlatformAdminUi\Form\Data\ContentType\Translation\TranslationAddData|null $data
+     * @param string|null $name
+     *
+     * @return \Symfony\Component\Form\FormInterface
+     */
+    public function addContentTypeTranslation(
```

Comment:
At this point we have to stop adding new forms to this `FormFactory`. It +1300 lines already :( Just create a `ContentTypeFormFactory`.